cirq-core/cirq/transformers/measurement_transformers_test.py

```diff
@@ -457,23 +491,23 @@ def test_diagram():
     cirq.testing.assert_has_diagram(
         deferred,
         """
-                   ┌────┐
-0: ─────────────────@───────X────────M('c')───
-                    │                │
-1: ─────────────────┼─@──────────────M────────
-                    │ │              │
-2: ─────────────────┼@┼──────────────M────────
-                    │││              │
-3: ─────────────────┼┼┼@─────────────M────────
-                    ││││
-M('a', q=q(0)): ────X┼┼┼────M('a')────────────
-                     │││    │
-M('a', q=q(2)): ─────X┼┼────M─────────────────
-                      ││
-M('b', q=q(1)): ──────X┼────M('b')────────────
-                       │    │
-M('b', q=q(3)): ───────X────M─────────────────
-                   └────┘
+                      ┌────┐
+0: ────────────────────@───────X────────M('c')───
+                       │                │
+1: ────────────────────┼─@──────────────M────────
+                       │ │              │
+2: ────────────────────┼@┼──────────────M────────
+                       │││              │
+3: ────────────────────┼┼┼@─────────────M────────
+                       ││││
+M('a[0]', q=q(0)): ────X┼┼┼────M('a')────────────
+                        │││    │
+M('a[0]', q=q(2)): ─────X┼┼────M─────────────────
+                         ││
+M('b[0]', q=q(1)): ──────X┼────M('b')────────────
+                          │    │
+M('b[0]', q=q(3)): ───────X────M─────────────────
+                      └────┘
```

Comment:
Changing diagrams is allowed as per our backwards compatibility guidelines, so this change is fine. But we should update the PR description to explicitly call out that the PR changes diagrams.